templates/default/netnode/netnodeevents.html

```diff
@@ -0,0 +1,39 @@
+<!--// $Id$ //-->
+<table class="lmsbox lms-ui-sortable lms-ui-tab-container" id="netnodeevents">
```

Comment:
dodaj do tego `data-label="...."` i nie będzie potrzebne:

```js
    if (getCookie('netnodeeventspanel') == '1') {
        $('#netnodeeventspanel').show();
    }
```

Reply:
https://github.com/chilek/lms/pull/2273/commits/03ab161ba7bcad072e06910e032df28cfa5197ca

Reply:
i tutaj też to można zastosować https://github.com/chilek/lms/pull/2273/commits/ff2202b6c22a4fbf1733ba421813ce563b1afdaa